test/D2L.Security.OAuth2.WebApi.IntegrationTests/TestWebService/Controllers/AuthorizationAttributeTestsController.cs

```diff
@@ -7,39 +7,48 @@ namespace D2L.Security.OAuth2.TestWebService.Controllers {
 	public sealed class AuthorizationAttributeTestsController : ApiController {
 		[HttpGet]
 		[Authentication( services: true )]
-		[Route("authorization/unspecifiedscope")]
+		[Route( "authorization/unspecifiedscope" )]
 		public void UnspecifiedScope() {
 			// Will crash because we forgot a [RequireScope(...)]	
 		}
 
 		[HttpGet]
-		[RequireScope("foo", "bar", "baz")]
-		[Route("authorization/unspecifiedprincipaltype")]
+		[RequireScope( "foo", "bar", "baz" )]
+		[Route( "authorization/unspecifiedprincipaltype" )]
 		public void UnspecifiedPrincipalType() {
 			// Will crash because we forgot an [AllowFrom(...)]	
 		}
 
 		[HttpGet]
-		[RequireScope("foo","bar","baz")]
+		[RequireScope( "foo", "bar", "baz" )]
 		[Authentication( users: true )]
-		[Route("authorization/basic")]
+		[Route( "authorization/basic" )]
 		public void Basic() {
-			
+
 		}
 
 		[HttpGet]
 		[NoRequiredScope]
 		[Authentication( users: true, services: true )]
-		[Route("authorization/noscope")]
+		[Route( "authorization/noscope" )]
 		public void NoScope() {
-			
+
 		}
 
 		[HttpGet]
 		[AllowAnonymous]
-		[Route("authorization/anonymous")]
+		[Route( "authorization/anonymous" )]
 		public void Anonymous() {
-			
+
+		}
+
+		[HttpGet]
+		[NoImpersonation]
+		[NoRequiredScope]
+		[Authentication( users: true, services: true )]
+		[Route( "authorization/imp" )]

```

Comment:
This route doesn't allow impersonation, but `authorization/imp` might come off as the opposite.